### DISCUSSION TITLE
Unable to submit application materials and check application status

### DISCUSSION THREAD

**Discussion by GrooveSino** [community/community#70470] [GitHub Education]
### Hi everyone,

on the Upload proof page，I submitted the proof materials  and clicked '"process my application", after the progress bar was loaded，web page refreshed, the upload page returned and no prompts appeared(I submitted materials a month ago and was told there were insufficient materials, I know what that interface would look like if unsuccessful).

But I noticed that the nunber in URL(like .../discount_requests/**[there]**/additional_information) changed.

Also I can't find any status appear in this page.

This trouble has been there about a week, I've changed a computer and network to apply, but it still doesn't work.

**Comment by dvtkrlbs** [comment]
Same issue is happening to me I've been trying for the last week.

**Reply by GrooveSino** [reply]
Have you solved it now？

**Reply by dvtkrlbs** [reply]
no, now it refresh with the proof image still loaded and when I hit submit it says cant process your application now

**Reply by dvtkrlbs** [reply]
I solved it by uploading my transcript instead of student certification and it got instantly approved. I recommend uploading a different official document and screenshots work fine.

**Comment by 51n4k4r1m1** [comment]
I have this issue. I tried multiple times in some days but get same result. 
Let me explain with images, at first we have to take an image and upload it, after that we see this:

![image](https://github.com/community/community/assets/148292681/1f260d2b-02ae-45e8-824f-3fedce730caa)

by pressing the `Process my application`, a progress shows and some requests happens. 

![photo_2023-10-20_23-37-53](https://github.com/community/community/assets/148292681/7d4b6e6d-f044-4bb0-91d3-7e84df878023)

after that we redirect multiple times between `completed`, `application` and `school_information` so the last one redirected to `additional_information` and this response has the content of reset form page. In other words it shows a blank form without any value and without showing any error or warnings about the application.

![image](https://github.com/community/community/assets/148292681/4c6e9fa5-aa1a-47e7-b6c0-47eca77ea536)

So what's the problem? How could we submit and pass this stage?

**Reply by GrooveSino** [reply]
Have U solve it?

**Comment by ismeh** [comment]
Same problem, not solved yet

At the end of the [page](https://education.github.com/discount_requests/12364732/additional_information),"Process my application" is not working. When I press the button, it keep saying:

"Unable to submit your application. Please try again"

![imagen](https://github.com/community/community/assets/81519191/37b7c968-f388-4ebe-83e6-0daa0a1fcd48)
